libs/features/pages/about/src/lib/components/team-members-section/team-members-section.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+import { __highlightedTeamMembers } from '@elewa-website/data/sections';
+
+@Component({
+  selector: 'elewa-website-team-members-section',
+  templateUrl: './team-members-section.component.html',
+  styleUrls: ['./team-members-section.component.scss'],
+})
+export class TeamMembersSectionComponent {
+  highlightedTeamMembers = __highlightedTeamMembers;
```

Comment:
Hi @levy-web 
This needs to be an input, like so: 
@Input() TeamMember: []
This way we can pass the array of teamMembers objects as inputs from the parent.